### PR TITLE
Case Flags v2 Feature Branch

### PR DESCRIPTION
**Pull request by MateiHMCTS**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-8703
https://tools.hmcts.net/jira/browse/EUI-7243
https://tools.hmcts.net/jira/browse/EUI-7669

### Change description ###

Case Flags v2.1 
Reasonable Adjustments

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by Daniel-Lam — DISMISSED** [review]
@johnbenjamin-hmcts - all looks OK; approved

**Review by connorpgpmcelroy — DISMISSED** [review]
LGTM

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by connorpgpmcelroy — APPROVED** [review]
Lot of commented code but will approve

**Review by johnbenjamin-hmcts — DISMISSED** [review]

**Comment by Daniel-Lam** [comment]
> Lot of commented code but will approve

Thanks, @connorpgpmcelroy - I think I inherited that when I merged the latest from `master`

**Review by connorpgpmcelroy — DISMISSED** [review]
LGTM

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by connorpgpmcelroy — DISMISSED** [review]
LGTM

**Review by johnbenjamin-hmcts — DISMISSED** [review]

**Review by johnbenjamin-hmcts — DISMISSED** [review]

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

File: api/.env.defaults [hunk 1]
```diff
@@ -1,6 +1,6 @@
 NODE_CONFIG_DIR=../config
 ALLOW_CONFIG_MUTATIONS=1
-NODE_CONFIG_ENV=development
+NODE_CONFIG_ENV=demo
```
Comment (udaydenduluri33) [inline api/.env.defaults:3]: Can this be reverted?

Comment (Daniel-Lam) [reply api/.env.defaults:3]: @udaydenduluri33 - looks like this got changed via PR #2886 for EUI-7858; I'll revert it

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by johnbenjamin-hmcts — DISMISSED** [review]

**Review by udaydenduluri33 — APPROVED** [review]

**Review by johnbenjamin-hmcts — DISMISSED** [review]

**Review by udaydenduluri33 — APPROVED** [review]

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by udaydenduluri33 — APPROVED** [review]

**Review by johnbenjamin-hmcts — DISMISSED** [review]

**Review by udaydenduluri33 — DISMISSED** [review]

**Review by udaydenduluri33 — APPROVED** [review]

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by udaydenduluri33 — DISMISSED** [review]

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by udaydenduluri33 — APPROVED** [review]

**Review by johnbenjamin-hmcts — DISMISSED** [review]

**Review by johnbenjamin-hmcts — DISMISSED** [review]

**Review by DavidJayakumar — APPROVED** [review]

**Review by johnbenjamin-hmcts — APPROVED** [review]

**Review by DavidJayakumar — APPROVED** [review]

**Pull request merged**